core/src/main/java/org/apache/carbondata/core/datastore/compression/CompressorFactory.java

```diff
@@ -35,6 +35,7 @@
   private final Map<String, Compressor> allSupportedCompressors = new HashMap<>();
 
   public enum NativeSupportedCompressor {
+    NONE("none",NoneCompressor.class),
```

Comment:
add space after `,`

Reply:
suggest change to NONE("nocompress", NoneCompressor.class)
`nocompress` will be appended to the data file name